src/Symfony/Bundle/FrameworkBundle/Resources/config/translation_providers.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\Translation\Bridge\Crowdin\Provider\CrowdinProviderFactory;
+use Symfony\Component\Translation\Bridge\Loco\Provider\LocoProviderFactory;
+use Symfony\Component\Translation\Bridge\Lokalise\Provider\LokaliseProviderFactory;
+use Symfony\Component\Translation\Bridge\PoEditor\Provider\PoEditorProviderFactory;
+use Symfony\Component\Translation\Provider\NullProviderFactory;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('translation.provider_factory.null', NullProviderFactory::class)
+            ->args([
+                service('http_client'),
+                service('logger'),
+                param('kernel.default_locale'),
+            ])
+            ->tag('translation.provider_factory')
+
+        ->set('translation.provider_factory.crowdin', CrowdinProviderFactory::class)
+            ->args([
+                service('http_client'),
+                service('logger'),
+                param('kernel.default_locale'),
+                service('translation.loader.xliff'),
+                service('translation.dumper.xliff'),
```

Comment:
As the recommended best practice is to use XLIFF, and this feature targets to automates the management of translation content, I think we should promote XLIFF, even enforce it here. 
It may be possible that a future provider will not support XLIFF (but I highly doubt it, as it is *the* OASIS standard, and it used also by Apple for iOS translation files). 
I think we must limit to XLIFF, and discuss later, it the case happen, to remove this limit.